packages/@aws-cdk/aws-apigateway/test/integ.stepFunctions-api.deploymentStack.ts

```diff
@@ -0,0 +1,33 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as apigw from '../lib';
+
+class StepFunctionsRestApiDeploymentStack extends cdk.Stack {
```

Comment:
Can you add a set of steps that can be used to verify this stack after deployment? see https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests

This will help future contributors test that this works when this is being changed.